mlflow/gluon.py

```diff
@@ -187,25 +193,28 @@ def log_model(gluon_model, artifact_path, conda_env=None):
                             ]
                         }
 
-    >>> from mxnet.gluon import Trainer
-    >>> from mxnet.gluon.contrib import estimator
-    >>> from mxnet.gluon.loss import SoftmaxCrossEntropyLoss
-    >>> from mxnet.gluon.nn import HybridSequential
-    >>> from mxnet.metric import Accuracy
-    >>> import mlflow
-    >>> # Build, compile, and train your model
-    >>> net = HybridSequential()
-    >>> with net.name_scope():
-    >>> ...
-    >>> net.hybridize()
-    >>> net.collect_params().initialize()
-    >>> softmax_loss = SoftmaxCrossEntropyLoss()
-    >>> trainer = Trainer(net.collect_params())
-    >>> est = estimator.Estimator(net=net, loss=softmax_loss, metrics=Accuracy(), trainer=trainer)
-    >>> # Log metrics and log the model
-    >>> with mlflow.start_run() as run:
-    >>>   est.fit(train_data=train_data, epochs=100, val_data=validation_data)
-    >>>   mlflow.gluon.log_model(net, "model")
+    .. code-block:: python
+        :caption: Example
+
+        from mxnet.gluon import Trainer
+        from mxnet.gluon.contrib import estimator
+        from mxnet.gluon.loss import SoftmaxCrossEntropyLoss
+        from mxnet.gluon.nn import HybridSequential
+        from mxnet.metric import Accuracy
+        import mlflow
+        # Build, compile, and train your model
+        net = HybridSequential()
+        with net.name_scope():
+            ...
+        net.hybridize()
+        net.collect_params().initialize()
+        softmax_loss = SoftmaxCrossEntropyLoss()
+        trainer = Trainer(net.collect_params())
+        est = estimator.Estimator(net=net, loss=softmax_loss, metrics=Accuracy(), trainer=trainer)
+        # Log metrics and log the model
+        with mlflow.start_run() as run:
```

Comment:
```suggestion
        with mlflow.start_run():
```

Remove `as run` because `run` is not used?

Reply:
Sure, if you not going to using a variable in the context manager block, as any place elsewhere, you don't need it. 

Will remove it.

Reply:
fixed!